pyvista/__init__.py

```diff
@@ -9,23 +9,239 @@
 # Load default theme.  Must be loaded first.
 from pyvista._version import __version__
 from pyvista.plotting import *
-from pyvista.utilities import *
-from pyvista.core import *
+from pyvista.utilities import (
+    # arrays,
+    # cells,
+    # common,
+    # errors,
+    # features,
+    fileio,
+    # geometric_objects,
+    # helpers,
+    # misc,
+    # parametric_objects,
+    reader,
+    # regression,
+    # sphinx_gallery,
+    transformations,
+    # wrappers,
+    # xvfb,
+)
```

Comment:
I've only kept subpackages here that broke our own CI. Note that if we delete things from here it should only affect
```py
import pyvista
pyvista.misc.something
```
whereas `pyvista.utilities.misc.something` could still work, especially if one does `from pyvista.utilities.misc import something`.